ngx-fudis/projects/ngx-fudis/src/lib/types/spacing.ts

```diff
@@ -17,9 +17,9 @@ export type FudisSpacingResponsive = {
 export const defaultSpacingValue = '0';
 
 /**
- * Spacing tokens
- * TODO: Would be better to not allow string type since we only want to allow spacing directive to take in predefined spacing token values.
- * This affects in breakpoint-utils.ts when pushing new spacing value to FudisBreakpointStyleResponsive array
+ * Spacing tokens TODO: Would be better to not allow string type since we only want to allow spacing
```

Comment:
My suggestion is to take the TODO comment out of the JSDoc and place it above it with basic comment out syntax `// TODO: ...` because our internal TODOs probably shouldn't be included in any JSDocs.